src/reference/00-Getting-Started/07A-Scopes.md

```diff
@@ -379,104 +379,153 @@ lazy val util = (project in file("util"))
 #### Delegation in detail (.value lookup)
 
 Now that we've covered all the details of scoping, we can explain the `.value`
-lookup in detail. It's ok to skip this section at in the beginning.
+lookup in detail.
 To summarize what we've learned so far:
 
-- A particular scope consists of three scope axes: Subprojects, Dependency configuration, and Tasks.
-  You can think of axes like as digits in a number, or (r, g, b) components in a color.
-- There's a special value `Global` scope (written as `*` in shell) that can be applied to either of the axes.
-- There's a special value `ThisBuild` scope (written as `{.}` in shell) only for Subprojects axis.
-- build.sbt can reference a scoped key using `.in(...)` method.
+- Scopes have three axes: a subproject axis, a dependency configuration axis, and a task axis.
+- There's a special value `Global` (written as `*` in shell) for any of the scope axes.
```

Comment:
Is it necessary to say how it's written in the shell here? I feel it belongs to another section. Also, I've also felt that these symbols are very hard to parse and internalize. I think that it would help to replace `{.}` by `ThisBuild` and`*` by `Global`.